doc/030_preparing_a_new_repo.rst

```diff
@@ -381,6 +381,48 @@ established.
 .. _service account: https://cloud.google.com/storage/docs/authentication#service_accounts
 .. _create a service account key: https://cloud.google.com/storage/docs/authentication#generating-a-private-key
 
+Google Drive
+************
+
+Restic supports Google Drive as a backend.
+
+Restic connects to Google Drive via a `service account`_.
+
+To use the Google Drive backend, first `create a service account key`_
+and download the JSON credentials file to ``~/.config/restic/gdrive-account-key.json``.
```

Comment:
At the moment, we're using environment variables to specify such things, can you please implement that similar to the [GCS backend](https://github.com/restic/restic/blob/5f43e179185ecb96819bfb6301746b5b4be7dec1/cmd/restic/global.go#L441)?